manager/src/test/java/com/redhat/service/smartevents/manager/ProcessorServiceTest.java

```diff
@@ -9,533 +8,435 @@
     }
 
     @Test
-    public void getProcessors() {
-        Bridge b = createPersistBridge(ManagedResourceStatus.READY);
-        ProcessorRequest r = new ProcessorRequest("My Processor", createKafkaAction());
-
-        Processor processor = processorService.createProcessor(b.getId(), b.getCustomerId(), r);
-        TestUtils.waitForProcessorDependenciesToBeReady(processorDAO, processor);
-
-        ListResult<Processor> results = processorService.getProcessors(b.getId(), TestConstants.DEFAULT_CUSTOMER_ID, new QueryInfo(0, 100));
+    void getProcessors() {
+        ListResult<Processor> results = processorService.getProcessors(DEFAULT_BRIDGE_ID, DEFAULT_CUSTOMER_ID, QUERY_INFO);
         assertThat(results.getPage()).isZero();
-        assertThat(results.getSize()).isEqualTo(1L);
-        assertThat(results.getTotal()).isEqualTo(1L);
-
-        assertThat(results.getItems().get(0).getId()).isEqualTo(processor.getId());
+        assertThat(results.getSize()).isEqualTo(3L);
+        assertThat(results.getTotal()).isEqualTo(3L);
+        assertThat(results.getItems().get(0).getId()).isEqualTo(DEFAULT_PROCESSOR_ID);
+        assertThat(results.getItems().get(1).getId()).isEqualTo(PROVISIONING_PROCESSOR_ID);
+        assertThat(results.getItems().get(2).getId()).isEqualTo(FAILED_PROCESSOR_ID);
     }
 
     @Test
-    public void getProcessors_noProcessorsOnBridge() {
-        Bridge b = createPersistBridge(ManagedResourceStatus.READY);
-        ListResult<Processor> results = processorService.getProcessors(b.getId(), TestConstants.DEFAULT_CUSTOMER_ID, new QueryInfo(0, 100));
+    void getProcessors_noProcessorsOnBridge() {
+        when(processorDAO.findByBridgeIdAndCustomerId(eq(DEFAULT_BRIDGE_ID), eq(DEFAULT_CUSTOMER_ID), any()))
+                .thenReturn(new ListResult<>(Collections.emptyList(), 0, 0));
+
+        ListResult<Processor> results = processorService.getProcessors(DEFAULT_BRIDGE_ID, DEFAULT_CUSTOMER_ID, QUERY_INFO);
         assertThat(results.getPage()).isZero();
         assertThat(results.getSize()).isZero();
         assertThat(results.getTotal()).isZero();
     }
 
     @Test
-    public void getProcessors_bridgeDoesNotExist() {
-        assertThatExceptionOfType(ItemNotFoundException.class).isThrownBy(() -> processorService.getProcessors("doesNotExist", TestConstants.DEFAULT_CUSTOMER_ID, new QueryInfo(0, 100)));
+    void getProcessors_bridgeDoesNotExist() {
+        assertThatExceptionOfType(ItemNotFoundException.class)
+                .isThrownBy(() -> processorService.getProcessors(NON_EXISTING_BRIDGE_ID, DEFAULT_CUSTOMER_ID, QUERY_INFO));
     }
 
     @Test
-    public void deleteProcessor_processorDoesNotExist() {
-        Bridge b = createPersistBridge(ManagedResourceStatus.READY);
-        assertThatExceptionOfType(ItemNotFoundException.class).isThrownBy(() -> processorService.deleteProcessor(b.getId(), "doesNotExist", b.getCustomerId()));
+    void getProcessorsCount() {
```

Comment:
It's a bit weird having a `getXXX()` method returning `void` that does an assertion.

Reply:
Yes I agree. I prefixed all the test methods with `"test"` to avoid confusion.